src/main/java/xyz/groundx/caver_ext_kas/kas/wallet/Wallet.java

```diff
@@ -2426,52 +2431,52 @@ public AccountCountByKRN getAccountCountByKRN() throws ApiException {
      * @throws ApiException
      */
     public Call getAccountCountByKRNAsync(ApiCallback<AccountCountByKRN> callback) throws ApiException {
-        return getAccountCountByKRNAsync(null, callback);
-    }
-
-    /**
-     * Return the number of accounts by passed as KRN in KAS.<br>
-     * GET /v2/stat/count/krn
-     *
-     * <pre>Example:
-     * {@code
-     * String krn = "{krn}";
-     * AccountCountByKRN accountCountByKRN = caver.kas.wallet.getAccountCountByKRN(krn);
-     * }
-     * </pre>
-     *
-     * @param krn The krn string to search
-     * @return AccountCountByKRN
-     * @throws ApiException
-     */
-    public AccountCountByKRN getAccountCountByKRN(String krn) throws ApiException {
-        return getStatisticsApi().getAccountCountByKRN(getChainId(), krn);
-    }
-
-    /**
-     * Return the number of accounts by passed as KRN in KAS asynchronously.<br>
-     * GET /v2/stat/count/krn
-     *
-     * <pre>Example:
-     * {@code
-     * String krn = "{krn}";
-     *
-     * ApiCallback<AccountCountByKRN> callback = new ApiCallback<AccountCountByKRN>() {
-     *     ....implements callback method
-     * };
-     *
-     * caver.kas.wallet.getAccountCountByKRNAsync(krn, callback);
-     * }
-     * </pre>
-     *
-     * @param krn The krn string to search
-     * @param callback The callback function to handle response.
-     * @return AccountCountByKRN
-     * @throws ApiException
-     */
-    public Call getAccountCountByKRNAsync(String krn, ApiCallback<AccountCountByKRN> callback) throws ApiException {
-        return getStatisticsApi().getAccountCountByKRNAsync(getChainId(), krn, callback);
-    }
+        return getStatisticsApi().getAccountCountByKRNAsync(getChainId(), callback);
+    }
+
+//    /**
+//     * Return the number of accounts by passed as KRN in KAS.<br>
+//     * GET /v2/stat/count/krn
+//     *
+//     * <pre>Example:
+//     * {@code
+//     * String krn = "{krn}";
+//     * AccountCountByKRN accountCountByKRN = caver.kas.wallet.getAccountCountByKRN(krn);
+//     * }
+//     * </pre>
+//     *
+//     * @param krn The krn string to search
+//     * @return AccountCountByKRN
+//     * @throws ApiException
+//     */
+//    public AccountCountByKRN getAccountCountByKRN() throws ApiException {
+//        return getStatisticsApi().getAccountCountByKRN(getChainId());
```

Comment:
```suggestion
//    public AccountCountByKRN getAccountCountByKRN(String krn) throws ApiException {
//        return getStatisticsApi().getAccountCountByKRN(getChainId(), krn);
```
If this comments is for future usage, then maybe in there `krn` parameter is missing?